apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -61,6 +62,8 @@
    */
   static final String BLUEBUTTON_STU3_RESOURCE_PROVIDERS = "bluebuttonStu3ResourceProviders";
 
+  static final String BLUEBUTTON_R4_RESOURCE_PROVIDERS = "bluebuttonR4ResourceProviders";
```

Comment:
I'd stick with the "v1" and "v2" naming here, too.

Reply:
We are talking of FHIR resources, so STU3, R4, R5 etc. V1, V2, V3 etc will facade the specific FHIR and HAPI version (or any other FHIR framework) for external partners.

Reply:
I have renamed the FHIR servers as v1 and v2 from STU3 and R4.